lib/services/watcher.go

```diff
@@ -961,13 +956,20 @@ func NewCertAuthorityWatcher(ctx context.Context, cfg CertAuthorityWatcherConfig
 
 	collector := &caCollector{
 		CertAuthorityWatcherConfig: cfg,
+		fanout:                     NewFanout(),
+		cas:                        make(map[types.CertAuthType]map[string]types.CertAuthority, len(cfg.Types)),
```

Comment:
nit:
```suggestion
		cas:                        make(types.CertAuthorityTypeMap, len(cfg.Types)),
```
I also don't mind renaming the `CertAuthorityTypeMap` if you have any better idea.

Reply:
I don't mind removing them, just `map[types.CertAuthType]map[string]types.CertAuthority` looks a bit "scary" in my opinion.